### ISSUE TITLE
(A11y severity 3) Add focus indicators

### ISSUE THREAD

**Issue by sathomas** [oaeproject/3akai-ux#3933]
Several elements within the site lack focus indicators. When navigating through the items using the keyboard, there is no visual indicator of where the user is on the page. This is generally caused by the presence of `outline:0` styles being applied to links and other navigable controls (see http://webaim.org/blog/plague-of-outline-0/). If this style is being applied, it should be removed from these controls. Instances of `a:hover{…}` (e.g., links that become underlined when the user hovers over them with a mouse) should generally be changed to `a:hover, a:focus {…}`. This will present the hover styles to keyboard users when they navigate to the element.
